src/sass/extends/_cards.scss

```diff
@@ -642,6 +642,11 @@ Cards - Table of Contents
   .card.card-foldable {
     border-bottom: 1px solid $uds-color-base-gray-3;
   }
+  .card-header .card-icon {
```

Comment:
Small correction to icon cards. CSS only, shouldn't require a fix for the UDS-Card block.